CONTRIBUTING.md

```diff
@@ -9,6 +9,25 @@
 1. `terraform plan`
 1. `terraform apply`
 
+# Debugging
+
+If terraform still uses the preinstalled version you can try to move the binary to the explicit plugins folder, as mentioned in the [official terraform documentation](https://developer.hashicorp.com/terraform/plugin/debugging) or in a [terraform tutorial](https://developer.hashicorp.com/terraform/tutorials/providers/provider-debug):
+```bash
+mv terraform-provider-virtualbox ~/.terraform.d/plugins/registry.local/local/virtualbox/5.0.0/linux_amd64/terraform-provider-virtualbox_v5.0.0
```

Comment:
Is it possible to do a symlink instead? It would result in a faster iteration cycle rather than having to build + mv.

Reply:
The documentation suggests to move it, but I think a symlink should also work.

Reply:
@VoyTechnology I've tested it now and a symlink does also work. I've added a simple note to the `CONTRIBUTING.md`.